sites/footers/footers-en.html

```diff
@@ -1,48 +1,195 @@
 ---
 {
-	"title": "Content page including complete footer",
+	"title": "Footer",
 	"language": "en",
 	"altLangPage": "footers-fr.html",
 	"secondlevel": false,
-	"dateModified": "2022-05-18",
-	"share": "true"
+	"dateModified": "2022-06-20",
+	"share": "true",
+	"contextualFooter": {
+		"title": "Page-specific contextual footer title",
+		"links": [
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 1"},
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 2"},
+			{ "url": "http://canada.ca/en", "text": "Page-specific Link 3"}
+		]
+	}
 }
 ---
-<p>Content page presenting complete default footer including the "About gvernment" section and all utility links from "About this site" section.</p>
-<p>Updated to use the <code>colcount-xs-2</code> class instead of <code>colcount-sm-2</code>.</p>
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Version 3.0</h2>
+<p>The following changes have been made since version 2.0:</p>
+<ul>
+	<li>Added a "Contextual" section above the "About government" section.
+		<ul>
+			<li>The title and links are defined globally in <code>_config.yml</code>.</li>
+			<li>The title and links can be changed for specific pages in the page's <code>contextualFooter</code> property.</li>
+		</ul>
+	</li>
+	<li>Added new heading "Government of Canada" in the "About government" section.</li>
+	<li>Added row of government-wide links.</li>
+	<li>Added short white horizontal rule (<code>&lt;hr /&gt;</code>) to separate government-wide links and theme links.</li>
+	<li>Modified the list of theme links.</li>
+	<li>The classes for the thematic links list were updated to <code>list-col-xs-1 list-col-sm-2 list-col-md-3</code>.
+		<ul>
+			<li>The columns now use the CSS "Flexbox" feature which distributes links horizontally instead of vertically.</li>
+			<li>Links are now displayed in one (1) column on extra small screens, in two (2) columns on small screens, and in three (3) columns on medium screens and up.</li>
+		</ul>
+	</li>
+	<li>The <code>padding</code> of existing <code>nav</code> elements has been slightly decreased.</li>
+</ul>
+<p>Working example: <a href="#wb-info">Scroll to the bottom of this page</a></p>
 
-<h2>Expected output code</h2>
+<h3>Expected output code</h3>
 <pre><code>&lt;footer id="wb-info"&gt;
+	&lt;div class="contextual"&gt;
+		&lt;nav class="container"&gt;
+			&lt;h3&gt;Contextual footer title&lt;/h3&gt;
```

Comment:
To match title defined:
```suggestion
			&lt;h3&gt;[Contextual footer title]&lt;/h3&gt;
```
(Similar changes needed in French.)